test/UserTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Icepace\User;
+use PHPUnit\Framework\TestCase;
+require_once 'vendor/autoload.php';
+
+class UserTest extends TestCase
+{
+    public function testSuccessCreateUserCardHtml1()
+    {
+        $user = new User(1,'fakeUsername', 'fakeHashedPass', 'fake bio', 'fakeAvatar.jpeg');
+        $result = $user->createUserCardHtml();
+        $this->assertEquals('<div class="userCard"><img class="avatarImg" src="fakeAvatar.jpeg" alt="Profile Picture"><p class="' . "cardUsernameText" . '">fakeUsername</p></div>', $result);
+    }
+
+    public function testSuccessCreateUserCardHtml2()
+    {
+        $user = new User(1, 5, 'fakeHashedPass', 'fake bio', 3);
+        $result = $user->createUserCardHtml();
+        $this->assertEquals('<div class="userCard"><img class="avatarImg" src="3" alt="Profile Picture"><p class="' . "cardUsernameText" . '">5</p></div>', $result);
+    }
```

Comment:
this test isn't really testing a different execution of the method - the method has done exactly the same thing but with different valid inputs. so for that reason in my opinion there is no need for this second test

Reply:
removed